hw2/Views/LeagueView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  LeagueView.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class LeagueView: BaseView {
+    
+    private let logoImageView = UIImageView()
+    private let countryLabel = UILabel()
+    private let arrowImageView = UIImageView()
+    private let nameLabel = UILabel()
+        
+    override func addViews() {
+        super.addViews()
```

Comment:
Ne moraš zvati super. Jer nije zamišljeno da on išta sadržava.
 // Default implementation is empty

Reply:
.